praetorian_cli/ui/aegis/commands/job_helpers.py

```
@@ -0,0 +1,340 @@
+def interactive_capability_picker(menu, suggested=None):
+    """Interactive capability picker with fuzzy search."""
+    colors = getattr(menu, 'colors', DEFAULT_COLORS)
+
+    if suggested:
+        # Validate the suggested capability first
+        capability_info = menu.sdk.aegis.validate_capability(suggested)
+        if capability_info:
+            # Show the suggested capability and ask for confirmation
+            desc = (capability_info.get('description', '') or '')[:60]
+            target = capability_info.get('target', 'asset')
+            menu.console.print(f"\n  Suggested capability:")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)